test/bdd/pkg/context/context.go

```diff
@@ -69,6 +69,7 @@ const (
 // BDDContext is a global context shared between different test suites in bddtests
 type BDDContext struct {
 	EDVClient                  *edvclient.Client
+	EDVVaultID                 string
```

Comment:
Vault ID is for sure an EDV concept and isn't ambiguous with anything else here, so you don't need the `EDV` in the name `EDVVaultID`

Reply:
fixed